src/Components/Input/index.js

```diff
@@ -0,0 +1,14 @@
+import React, { Component } from 'react';
+
+import './index.css';
+
+export default class Input extends Component {
+    render() {
+        return (
+            <div className="InputContainer">
+                <label className="Input--InputLabel">{this.props.inputLabel}</label>
+                <input className="InputField" type={this.props.inputType} />
```

Comment:
What about removing `input` from the name of the props? So instead of `this.props.inputType` and `this.props.inputLabel` we would have `this.props.type` and `this.props.label` ? I think we can do this because we already know this component is an input so I think we can skip the redundancy. What's your thoughts?

Reply:
I had this doubt as well naming props I used. Is it okay to be this generic? Can't this.props.type be used somewhere else and be confusing?